static/src/js/components/CollectionSelector/__tests__/CollectionSelector.test.jsx

```diff
@@ -11,6 +11,7 @@ import userEvent from '@testing-library/user-event'
 
 import { GET_PERMISSION_COLLECTIONS } from '@/js/operations/queries/getPermissionCollections'
 
+import AppContext from '@/js/context/AppContext'
```

Comment:
When we import files us `@` syntax into files that have imports not using it, we should be taking that opportunity to update the existing imports to use the newer `@` syntax.

Reply:
done